bincrafters/autodetect.py

```diff
@@ -93,13 +93,17 @@ def autodetect() -> str:
 DIR_STRUCTURE_ONE_RECIPE_ONE_VERSION = "one_recipe_one_file"
 DIR_STRUCTURE_ONE_RECIPE_MANY_VERSIONS = "one_recipe_many_versions"
 DIR_STRUCTURE_CCI = "cci_many_recipes_many_versions"
+DIR_STRUCTURE_STANDALONE_RECIPE_MANY_VERSIONS = "standalone_recipe_many_versions"
 
 
 def autodetect_directory_structure() -> str:
     """ Return the directory type as classified above this method
     """
     pwd = os.getcwd()
 
+    if os.path.exists(os.path.join(pwd, "conanfile.py")) and os.path.exists(os.path.join(pwd, "conandata.yml")):
+        return DIR_STRUCTURE_STANDALONE_RECIPE_MANY_VERSIONS
```

Comment:
This breaks existing behaviour as there is no way to differentiate between this and `DIR_STRUCTURE_ONE_RECIPE_ONE_VERSION`. In both cases, there is a `conanfile.py` and in the case of `DIR_STRUCTURE_ONE_RECIPE_ONE_VERSION` there CAN be a `conandata.yml` with several versions.